Build is broken

Hi! Thank you very much for providing this image. Unfortunately it seems to be broken.

    % podman --version
    podman version 3.4.7


```
% podman build --no-cache -t intel-mkl:2020 .
[1/3] STEP 1/6: FROM nobodyxu/apt-fast:latest-debian-buster-slim AS install-mkl
[1/3] STEP 2/6: RUN apt-auto install -y --no-install-recommends apt-transport-https ca-certificates gnupg2 gnupg-agent                                                 software-properties-common curl apt-utils
Get:1 http://security.debian.org/debian-security buster/updates InRelease [65.4 kB]
Get:2 http://deb.debian.org/debian buster InRelease [122 kB]
Get:3 http://deb.debian.org/debian buster-updates InRelease [51.9 kB]
Reading package lists...
E: Repository 'http://security.debian.org/debian-security buster/updates InRelease' changed its 'Suite' value from 'stable' to 'oldstable'
E: Repository 'http://deb.debian.org/debian buster InRelease' changed its 'Suite' value from 'stable' to 'oldstable'
E: Repository 'http://deb.debian.org/debian buster-updates InRelease' changed its 'Suite' value from 'stable-updates' to 'oldstable-updates'
[apt-fast 10:04:54]
[apt-fast 10:04:54]Working... this may take a while.
[apt-fast 10:04:54]

05/13 10:04:56 [NOTICE] Downloading 61 item(s)

05/13 10:04:56 [ERROR] CUID#7 - Download aborted. URI=http://security.debian.org/debian-security/pool/updates/main/o/openldap/libldap-common_2.4.47+dfsg-3+deb10u3_all.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://security.debian.org/debian-security/pool/updates/main/o/openldap/libldap-common_2.4.47+dfsg-3+deb10u3_all.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:56 [ERROR] CUID#8 - Download aborted. URI=http://security.debian.org/debian-security/pool/updates/main/o/openldap/libldap-2.4-2_2.4.47+dfsg-3+deb10u3_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://security.debian.org/debian-security/pool/updates/main/o/openldap/libldap-2.4-2_2.4.47+dfsg-3+deb10u3_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:56 [ERROR] CUID#9 - Download aborted. URI=http://deb.debian.org/debian/pool/main/p/python3.7/libpython3.7-minimal_3.7.3-2+deb10u2_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/p/python3.7/libpython3.7-minimal_3.7.3-2+deb10u2_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:56 [ERROR] CUID#10 - Download aborted. URI=http://deb.debian.org/debian/pool/main/e/expat/libexpat1_2.2.6-2+deb10u1_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/e/expat/libexpat1_2.2.6-2+deb10u1_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:56 [ERROR] CUID#11 - Download aborted. URI=http://deb.debian.org/debian/pool/main/p/python3.7/python3.7-minimal_3.7.3-2+deb10u2_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/p/python3.7/python3.7-minimal_3.7.3-2+deb10u2_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found
[DL:0B][#3e9787 0B/0B][#b40794 0B/0B][#a262bf 0B/0B][#d98add 0B/0B][#66fa04 0B/0B]

05/13 10:04:57 [NOTICE] Download GID#3e97871b67ae33c8 not complete: /var/cache/apt/apt-fast/libldap-common_2.4.47+dfsg-3+deb10u3_all.deb

05/13 10:04:57 [NOTICE] Download GID#b40794798dd20e30 not complete: /var/cache/apt/apt-fast/libldap-2.4-2_2.4.47+dfsg-3+deb10u3_amd64.deb

05/13 10:04:57 [NOTICE] Download GID#a262bf198cc07c5d not complete: /var/cache/apt/apt-fast/libpython3.7-minimal_3.7.3-2+deb10u2_amd64.deb

05/13 10:04:57 [NOTICE] Download GID#d98add4c0bf76d12 not complete: /var/cache/apt/apt-fast/libexpat1_2.2.6-2+deb10u1_amd64.deb

05/13 10:04:57 [NOTICE] Download GID#66fa046a871c0dd3 not complete: /var/cache/apt/apt-fast/python3.7-minimal_3.7.3-2+deb10u2_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libmpdec2_2.4.2-2_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libmpdec2_2.4.2-2_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libreadline7_7.0-5_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libreadline7_7.0-5_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/readline-common_7.0-5_all.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/readline-common_7.0-5_all.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/python3-minimal_3.7.3-1_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/python3-minimal_3.7.3-1_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/mime-support_3.62_all.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/mime-support_3.62_all.deb

05/13 10:04:57 [ERROR] CUID#27 - Download aborted. URI=http://deb.debian.org/debian/pool/main/p/python3.7/python3.7_3.7.3-2+deb10u2_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/p/python3.7/python3.7_3.7.3-2+deb10u2_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#b454f47496407a5d not complete: /var/cache/apt/apt-fast/python3.7_3.7.3-2+deb10u2_amd64.deb

05/13 10:04:57 [ERROR] CUID#33 - Download aborted. URI=http://deb.debian.org/debian/pool/main/c/cyrus-sasl2/libsasl2-modules-db_2.1.27+dfsg-1+deb10u1_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/c/cyrus-sasl2/libsasl2-modules-db_2.1.27+dfsg-1+deb10u1_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#0478b9d2a82b8ed7 not complete: /var/cache/apt/apt-fast/libsasl2-modules-db_2.1.27+dfsg-1+deb10u1_amd64.deb

05/13 10:04:57 [ERROR] CUID#34 - Download aborted. URI=http://deb.debian.org/debian/pool/main/c/cyrus-sasl2/libsasl2-2_2.1.27+dfsg-1+deb10u1_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/c/cyrus-sasl2/libsasl2-2_2.1.27+dfsg-1+deb10u1_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#5bfd884f19333ef2 not complete: /var/cache/apt/apt-fast/libsasl2-2_2.1.27+dfsg-1+deb10u1_amd64.deb

05/13 10:04:57 [ERROR] CUID#35 - Download aborted. URI=http://deb.debian.org/debian/pool/main/a/apt/apt-transport-https_1.8.2.1_all.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/a/apt/apt-transport-https_1.8.2.1_all.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#c72d8657f12d4d3a not complete: /var/cache/apt/apt-fast/apt-transport-https_1.8.2.1_all.deb

05/13 10:04:57 [ERROR] CUID#37 - Download aborted. URI=http://deb.debian.org/debian/pool/main/k/krb5/libkrb5support0_1.17-3_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/k/krb5/libkrb5support0_1.17-3_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#468977a9fbd9b326 not complete: /var/cache/apt/apt-fast/libkrb5support0_1.17-3_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libkeyutils1_1.6-6_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libkeyutils1_1.6-6_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/python3_3.7.3-1_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/python3_3.7.3-1_amd64.deb

05/13 10:04:57 [ERROR] CUID#38 - Download aborted. URI=http://deb.debian.org/debian/pool/main/k/krb5/libk5crypto3_1.17-3_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/k/krb5/libk5crypto3_1.17-3_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#4d54dcaad1866619 not complete: /var/cache/apt/apt-fast/libk5crypto3_1.17-3_amd64.deb

05/13 10:04:57 [ERROR] CUID#25 - Download aborted. URI=http://deb.debian.org/debian/pool/main/p/python3.7/libpython3.7-stdlib_3.7.3-2+deb10u2_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/p/python3.7/libpython3.7-stdlib_3.7.3-2+deb10u2_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#aed4696dd7828f98 not complete: /var/cache/apt/apt-fast/libpython3.7-stdlib_3.7.3-2+deb10u2_amd64.deb

05/13 10:04:57 [ERROR] CUID#40 - Download aborted. URI=http://deb.debian.org/debian/pool/main/k/krb5/libkrb5-3_1.17-3_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/k/krb5/libkrb5-3_1.17-3_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#2b740aa11e6caa6e not complete: /var/cache/apt/apt-fast/libkrb5-3_1.17-3_amd64.deb

05/13 10:04:57 [ERROR] CUID#42 - Download aborted. URI=http://deb.debian.org/debian/pool/main/k/krb5/libgssapi-krb5-2_1.17-3_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/k/krb5/libgssapi-krb5-2_1.17-3_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#e71d75b52313cd66 not complete: /var/cache/apt/apt-fast/libgssapi-krb5-2_1.17-3_amd64.deb

05/13 10:04:57 [ERROR] CUID#46 - Download aborted. URI=http://deb.debian.org/debian/pool/main/c/curl/libcurl4_7.64.0-4+deb10u1_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/c/curl/libcurl4_7.64.0-4+deb10u1_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#a4c205525c5ec4be not complete: /var/cache/apt/apt-fast/libcurl4_7.64.0-4+deb10u1_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/librtmp1_2.4+20151223.gitfa8646d.1-2_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/librtmp1_2.4+20151223.gitfa8646d.1-2_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libpython3-stdlib_3.7.3-1_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libpython3-stdlib_3.7.3-1_amd64.deb

05/13 10:04:57 [ERROR] CUID#47 - Download aborted. URI=http://deb.debian.org/debian/pool/main/c/curl/curl_7.64.0-4+deb10u1_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/c/curl/curl_7.64.0-4+deb10u1_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#0839359a7fcad984 not complete: /var/cache/apt/apt-fast/curl_7.64.0-4+deb10u1_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libnghttp2-14_1.36.0-2+deb10u1_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libnghttp2-14_1.36.0-2+deb10u1_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libpsl5_0.20.2-2_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libpsl5_0.20.2-2_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libassuan0_2.5.2-1_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libassuan0_2.5.2-1_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/lsb-base_10.2019051400_all.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/lsb-base_10.2019051400_all.deb

05/13 10:04:57 [ERROR] CUID#60 - Download aborted. URI=http://deb.debian.org/debian/pool/main/d/distro-info-data/distro-info-data_0.41+deb10u2_all.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/d/distro-info-data/distro-info-data_0.41+deb10u2_all.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#8630136b91619790 not complete: /var/cache/apt/apt-fast/distro-info-data_0.41+deb10u2_all.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libksba8_1.3.5-2_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libksba8_1.3.5-2_amd64.deb

05/13 10:04:57 [ERROR] CUID#61 - Download aborted. URI=http://deb.debian.org/debian/pool/main/g/glib2.0/libglib2.0-0_2.58.3-2+deb10u2_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/g/glib2.0/libglib2.0-0_2.58.3-2+deb10u2_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:04:57 [NOTICE] Download GID#05e76c8252b2d580 not complete: /var/cache/apt/apt-fast/libglib2.0-0_2.58.3-2+deb10u2_amd64.deb

05/13 10:04:57 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libnpth0_1.6-1_amd64.deb

05/13 10:04:57 [NOTICE] Download complete: /var/cache/apt/apt-fast/libnpth0_1.6-1_amd64.deb

05/13 10:04:58 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/dirmngr_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:58 [NOTICE] Download complete: /var/cache/apt/apt-fast/dirmngr_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:58 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libgirepository-1.0-1_1.58.3-2_amd64.deb

05/13 10:04:58 [NOTICE] Download complete: /var/cache/apt/apt-fast/libgirepository-1.0-1_1.58.3-2_amd64.deb

05/13 10:04:58 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gir1.2-packagekitglib-1.0_1.1.12-5_amd64.deb

05/13 10:04:58 [NOTICE] Download complete: /var/cache/apt/apt-fast/gir1.2-packagekitglib-1.0_1.1.12-5_amd64.deb

05/13 10:04:58 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gir1.2-glib-2.0_1.58.3-2_amd64.deb

05/13 10:04:58 [NOTICE] Download complete: /var/cache/apt/apt-fast/gir1.2-glib-2.0_1.58.3-2_amd64.deb

05/13 10:04:58 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libpackagekit-glib2-18_1.1.12-5_amd64.deb

05/13 10:04:58 [NOTICE] Download complete: /var/cache/apt/apt-fast/libpackagekit-glib2-18_1.1.12-5_amd64.deb
[DL:2.4MiB][#f5aeab 304KiB/497KiB(61%)][#4ca835 48KiB/0.9MiB(4%)][#e3bcbd 128KiB/841KiB(15%)][#f16eeb 16KiB/844KiB(1%)][#1f3bdd 0B/0B]

05/13 10:04:58 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/pinentry-curses_1.1.0-2_amd64.deb

05/13 10:04:58 [NOTICE] Download complete: /var/cache/apt/apt-fast/pinentry-curses_1.1.0-2_amd64.deb

05/13 10:04:58 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gnupg-utils_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:58 [NOTICE] Download complete: /var/cache/apt/apt-fast/gnupg-utils_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:59 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gpg_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:59 [NOTICE] Download complete: /var/cache/apt/apt-fast/gpg_2.2.12-1+deb10u1_amd64.deb
[DL:2.5MiB][#f5aeab 480KiB/497KiB(96%)][#4ca835 352KiB/0.9MiB(35%)][#cf35d0 304KiB/602KiB(50%)][#4c193f 208KiB/473KiB(43%)][#d90c3c 48KiB/466KiB(10%)]

05/13 10:04:59 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gpgconf_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:59 [NOTICE] Download complete: /var/cache/apt/apt-fast/gpgconf_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:59 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gpg-wks-client_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:59 [NOTICE] Download complete: /var/cache/apt/apt-fast/gpg-wks-client_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:59 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gpg-wks-server_2.2.12-1+deb10u1_amd64.deb

05/13 10:04:59 [NOTICE] Download complete: /var/cache/apt/apt-fast/gpg-wks-server_2.2.12-1+deb10u1_amd64.deb

05/13 10:05:00 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gpg-agent_2.2.12-1+deb10u1_amd64.deb

05/13 10:05:00 [NOTICE] Download complete: /var/cache/apt/apt-fast/gpg-agent_2.2.12-1+deb10u1_amd64.deb

05/13 10:05:00 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gnupg-agent_2.2.12-1+deb10u1_all.deb

05/13 10:05:00 [NOTICE] Download complete: /var/cache/apt/apt-fast/gnupg-agent_2.2.12-1+deb10u1_all.deb
[DL:2.6MiB][#4ca835 688KiB/0.9MiB(69%)][#e73d4b 256KiB/589KiB(43%)][#054d66 672KiB/698KiB(96%)][#030c2d 160KiB/383KiB(41%)][#cd7e02 176KiB/2.5MiB(6%)]

05/13 10:05:00 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gnupg_2.2.12-1+deb10u1_all.deb

05/13 10:05:00 [NOTICE] Download complete: /var/cache/apt/apt-fast/gnupg_2.2.12-1+deb10u1_all.deb

05/13 10:05:00 [ERROR] CUID#98 - Download aborted. URI=http://deb.debian.org/debian/pool/main/c/curl/libcurl3-gnutls_7.64.0-4+deb10u1_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/c/curl/libcurl3-gnutls_7.64.0-4+deb10u1_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:05:00 [NOTICE] Download GID#ae27c701562d6f67 not complete: /var/cache/apt/apt-fast/libcurl3-gnutls_7.64.0-4+deb10u1_amd64.deb

05/13 10:05:00 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gnupg2_2.2.12-1+deb10u1_all.deb

05/13 10:05:00 [NOTICE] Download complete: /var/cache/apt/apt-fast/gnupg2_2.2.12-1+deb10u1_all.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/libdbus-1-3_1.12.20-0+deb10u1_amd64.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/libdbus-1-3_1.12.20-0+deb10u1_amd64.deb

05/13 10:05:01 [ERROR] CUID#103 - Download aborted. URI=http://deb.debian.org/debian/pool/main/p/python-apt/python-apt-common_1.8.4.1_all.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/p/python-apt/python-apt-common_1.8.4.1_all.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:05:01 [NOTICE] Download GID#b4869110c7ca009d not complete: /var/cache/apt/apt-fast/python-apt-common_1.8.4.1_all.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/iso-codes_4.2-1_all.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/iso-codes_4.2-1_all.deb

05/13 10:05:01 [ERROR] CUID#104 - Download aborted. URI=http://deb.debian.org/debian/pool/main/p/python-apt/python3-apt_1.8.4.1_amd64.deb
Exception: [AbstractCommand.cc:351] errorCode=3 URI=http://deb.debian.org/debian/pool/main/p/python-apt/python3-apt_1.8.4.1_amd64.deb
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

05/13 10:05:01 [NOTICE] Download GID#2c1ac87a421b43e9 not complete: /var/cache/apt/apt-fast/python3-apt_1.8.4.1_amd64.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/lsb-release_10.2019051400_all.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/lsb-release_10.2019051400_all.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/python3-dbus_1.2.8-3_amd64.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/python3-dbus_1.2.8-3_amd64.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gnupg-l10n_2.2.12-1+deb10u1_all.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/gnupg-l10n_2.2.12-1+deb10u1_all.deb
[DL:2.9MiB][#e73d4b 448KiB/589KiB(75%)][#462c0a 144KiB/175KiB(82%)][#4d9c8a 32KiB/64KiB(49%)][#edb91f 0B/48KiB(0%)][#ffba4a 48KiB/81KiB(58%)]

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/python3-gi_3.30.4-1_amd64.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/python3-gi_3.30.4-1_amd64.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/python3-pycurl_7.43.0.2-0.1_amd64.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/python3-pycurl_7.43.0.2-0.1_amd64.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/python3-software-properties_0.96.20.2-2_all.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/python3-software-properties_0.96.20.2-2_all.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/software-properties-common_0.96.20.2-2_all.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/software-properties-common_0.96.20.2-2_all.deb

05/13 10:05:01 [NOTICE] Verification finished successfully. file=/var/cache/apt/apt-fast/gpgsm_2.2.12-1+deb10u1_amd64.deb

05/13 10:05:01 [NOTICE] Download complete: /var/cache/apt/apt-fast/gpgsm_2.2.12-1+deb10u1_amd64.deb

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
3e9787|ERR |       0B/s|/var/cache/apt/apt-fast/libldap-common_2.4.47+dfsg-3+deb10u3_all.deb
b40794|ERR |       0B/s|/var/cache/apt/apt-fast/libldap-2.4-2_2.4.47+dfsg-3+deb10u3_amd64.deb
a262bf|ERR |       0B/s|/var/cache/apt/apt-fast/libpython3.7-minimal_3.7.3-2+deb10u2_amd64.deb
d98add|ERR |       0B/s|/var/cache/apt/apt-fast/libexpat1_2.2.6-2+deb10u1_amd64.deb
66fa04|ERR |       0B/s|/var/cache/apt/apt-fast/python3.7-minimal_3.7.3-2+deb10u2_amd64.deb
c580c8|OK  |   1.2MiB/s|/var/cache/apt/apt-fast/libmpdec2_2.4.2-2_amd64.deb
c0e9a9|OK  |   1.5MiB/s|/var/cache/apt/apt-fast/libreadline7_7.0-5_amd64.deb
45f5f4|OK  |   774KiB/s|/var/cache/apt/apt-fast/readline-common_7.0-5_all.deb
f07c2f|OK  |   206KiB/s|/var/cache/apt/apt-fast/python3-minimal_3.7.3-1_amd64.deb
130fd6|OK  |   210KiB/s|/var/cache/apt/apt-fast/mime-support_3.62_all.deb
b454f4|ERR |       0B/s|/var/cache/apt/apt-fast/python3.7_3.7.3-2+deb10u2_amd64.deb
0478b9|ERR |       0B/s|/var/cache/apt/apt-fast/libsasl2-modules-db_2.1.27+dfsg-1+deb10u1_amd64.deb
5bfd88|ERR |       0B/s|/var/cache/apt/apt-fast/libsasl2-2_2.1.27+dfsg-1+deb10u1_amd64.deb
c72d86|ERR |       0B/s|/var/cache/apt/apt-fast/apt-transport-https_1.8.2.1_all.deb
468977|ERR |       0B/s|/var/cache/apt/apt-fast/libkrb5support0_1.17-3_amd64.deb
d85872|OK  |   1.5MiB/s|/var/cache/apt/apt-fast/libkeyutils1_1.6-6_amd64.deb
0e69f1|OK  |   1.2MiB/s|/var/cache/apt/apt-fast/python3_3.7.3-1_amd64.deb
4d54dc|ERR |       0B/s|/var/cache/apt/apt-fast/libk5crypto3_1.17-3_amd64.deb
aed469|ERR |       0B/s|/var/cache/apt/apt-fast/libpython3.7-stdlib_3.7.3-2+deb10u2_amd64.deb
2b740a|ERR |       0B/s|/var/cache/apt/apt-fast/libkrb5-3_1.17-3_amd64.deb
e71d75|ERR |       0B/s|/var/cache/apt/apt-fast/libgssapi-krb5-2_1.17-3_amd64.deb
a4c205|ERR |       0B/s|/var/cache/apt/apt-fast/libcurl4_7.64.0-4+deb10u1_amd64.deb
127786|OK  |   1.8MiB/s|/var/cache/apt/apt-fast/librtmp1_2.4+20151223.gitfa8646d.1-2_amd64.deb
b0e4b8|OK  |   170KiB/s|/var/cache/apt/apt-fast/libpython3-stdlib_3.7.3-1_amd64.deb
083935|ERR |       0B/s|/var/cache/apt/apt-fast/curl_7.64.0-4+deb10u1_amd64.deb
6c7238|OK  |   1.5MiB/s|/var/cache/apt/apt-fast/libnghttp2-14_1.36.0-2+deb10u1_amd64.deb
39a956|OK  |   1.1MiB/s|/var/cache/apt/apt-fast/libpsl5_0.20.2-2_amd64.deb
e8c575|OK  |   1.0MiB/s|/var/cache/apt/apt-fast/libassuan0_2.5.2-1_amd64.deb
052d90|OK  |   1.3MiB/s|/var/cache/apt/apt-fast/lsb-base_10.2019051400_all.deb
863013|ERR |       0B/s|/var/cache/apt/apt-fast/distro-info-data_0.41+deb10u2_all.deb
f45220|OK  |   824KiB/s|/var/cache/apt/apt-fast/libksba8_1.3.5-2_amd64.deb
05e76c|ERR |       0B/s|/var/cache/apt/apt-fast/libglib2.0-0_2.58.3-2+deb10u2_amd64.deb
9bca51|OK  |   100KiB/s|/var/cache/apt/apt-fast/libnpth0_1.6-1_amd64.deb
f97aff|OK  |   1.8MiB/s|/var/cache/apt/apt-fast/dirmngr_2.2.12-1+deb10u1_amd64.deb
0895b9|OK  |   708KiB/s|/var/cache/apt/apt-fast/libgirepository-1.0-1_1.58.3-2_amd64.deb
a891ca|OK  |   2.0MiB/s|/var/cache/apt/apt-fast/gir1.2-packagekitglib-1.0_1.1.12-5_amd64.deb
249956|OK  |   482KiB/s|/var/cache/apt/apt-fast/gir1.2-glib-2.0_1.58.3-2_amd64.deb
0eeb2c|OK  |   444KiB/s|/var/cache/apt/apt-fast/libpackagekit-glib2-18_1.1.12-5_amd64.deb
1f3bdd|OK  |   623KiB/s|/var/cache/apt/apt-fast/pinentry-curses_1.1.0-2_amd64.deb
e3bcbd|OK  |   1.1MiB/s|/var/cache/apt/apt-fast/gnupg-utils_2.2.12-1+deb10u1_amd64.deb
f16eeb|OK  |   0.9MiB/s|/var/cache/apt/apt-fast/gpg_2.2.12-1+deb10u1_amd64.deb
f5aeab|OK  |   297KiB/s|/var/cache/apt/apt-fast/gpgconf_2.2.12-1+deb10u1_amd64.deb
4c193f|OK  |   743KiB/s|/var/cache/apt/apt-fast/gpg-wks-client_2.2.12-1+deb10u1_amd64.deb
d90c3c|OK  |   0.9MiB/s|/var/cache/apt/apt-fast/gpg-wks-server_2.2.12-1+deb10u1_amd64.deb
cf35d0|OK  |   408KiB/s|/var/cache/apt/apt-fast/gpg-agent_2.2.12-1+deb10u1_amd64.deb
69cd64|OK  |   1.0MiB/s|/var/cache/apt/apt-fast/gnupg-agent_2.2.12-1+deb10u1_all.deb
054d66|OK  |   1.2MiB/s|/var/cache/apt/apt-fast/gnupg_2.2.12-1+deb10u1_all.deb
ae27c7|ERR |       0B/s|/var/cache/apt/apt-fast/libcurl3-gnutls_7.64.0-4+deb10u1_amd64.deb
030c2d|OK  |   441KiB/s|/var/cache/apt/apt-fast/gnupg2_2.2.12-1+deb10u1_all.deb
972830|OK  |   370KiB/s|/var/cache/apt/apt-fast/libdbus-1-3_1.12.20-0+deb10u1_amd64.deb
b48691|ERR |       0B/s|/var/cache/apt/apt-fast/python-apt-common_1.8.4.1_all.deb
cd7e02|OK  |   2.8MiB/s|/var/cache/apt/apt-fast/iso-codes_4.2-1_all.deb
2c1ac8|ERR |       0B/s|/var/cache/apt/apt-fast/python3-apt_1.8.4.1_amd64.deb
24b36b|OK  |   175KiB/s|/var/cache/apt/apt-fast/lsb-release_10.2019051400_all.deb
3ad7e9|OK  |   2.8MiB/s|/var/cache/apt/apt-fast/python3-dbus_1.2.8-3_amd64.deb
4ca835|OK  |   324KiB/s|/var/cache/apt/apt-fast/gnupg-l10n_2.2.12-1+deb10u1_all.deb
462c0a|OK  |   1.8MiB/s|/var/cache/apt/apt-fast/python3-gi_3.30.4-1_amd64.deb
4d9c8a|OK  |   719KiB/s|/var/cache/apt/apt-fast/python3-pycurl_7.43.0.2-0.1_amd64.deb
edb91f|OK  |   583KiB/s|/var/cache/apt/apt-fast/python3-software-properties_0.96.20.2-2_all.deb
ffba4a|OK  |   518KiB/s|/var/cache/apt/apt-fast/software-properties-common_0.96.20.2-2_all.deb
e73d4b|OK  |   298KiB/s|/var/cache/apt/apt-fast/gpgsm_2.2.12-1+deb10u1_amd64.deb

Status Legend:
(OK):download completed.(ERR):error occurred.

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.
Reading package lists...
Building dependency tree...
Reading state information...
apt-utils is already the newest version (1.8.2.1).
ca-certificates is already the newest version (20200601~deb10u1).
ca-certificates set to manually installed.
The following additional packages will be installed:
  dirmngr distro-info-data gir1.2-glib-2.0 gir1.2-packagekitglib-1.0 gnupg
  gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client gpg-wks-server gpgconf
  gpgsm iso-codes libassuan0 libcurl3-gnutls libcurl4 libdbus-1-3 libexpat1
  libgirepository-1.0-1 libglib2.0-0 libgssapi-krb5-2 libk5crypto3
  libkeyutils1 libkrb5-3 libkrb5support0 libksba8 libldap-2.4-2 libldap-common
  libmpdec2 libnghttp2-14 libnpth0 libpackagekit-glib2-18 libpsl5
  libpython3-stdlib libpython3.7-minimal libpython3.7-stdlib libreadline7
  librtmp1 libsasl2-2 libsasl2-modules-db lsb-base lsb-release mime-support
  pinentry-curses python-apt-common python3 python3-apt python3-dbus
  python3-gi python3-minimal python3-pycurl python3-software-properties
  python3.7 python3.7-minimal readline-common
Suggested packages:
  dbus-user-session libpam-systemd pinentry-gnome3 tor parcimonie xloadimage
  scdaemon isoquery krb5-doc krb5-user lsb pinentry-doc python3-doc python3-tk
  python3-venv python3-apt-dbg python-apt-doc python-dbus-doc python3-dbus-dbg
  libcurl4-gnutls-dev python-pycurl-doc python3-pycurl-dbg python3.7-venv
  python3.7-doc binutils binfmt-support readline-doc
Recommended packages:
  dbus libglib2.0-data shared-mime-info xdg-user-dirs krb5-locales packagekit
  publicsuffix libsasl2-modules bzip2 file xz-utils unattended-upgrades
The following NEW packages will be installed:
  apt-transport-https curl dirmngr distro-info-data gir1.2-glib-2.0
  gir1.2-packagekitglib-1.0 gnupg gnupg-agent gnupg-l10n gnupg-utils gnupg2
  gpg gpg-agent gpg-wks-client gpg-wks-server gpgconf gpgsm iso-codes
  libassuan0 libcurl3-gnutls libcurl4 libdbus-1-3 libexpat1
  libgirepository-1.0-1 libglib2.0-0 libgssapi-krb5-2 libk5crypto3
  libkeyutils1 libkrb5-3 libkrb5support0 libksba8 libldap-2.4-2 libldap-common
  libmpdec2 libnghttp2-14 libnpth0 libpackagekit-glib2-18 libpsl5
  libpython3-stdlib libpython3.7-minimal libpython3.7-stdlib libreadline7
  librtmp1 libsasl2-2 libsasl2-modules-db lsb-base lsb-release mime-support
  pinentry-curses python-apt-common python3 python3-apt python3-dbus
  python3-gi python3-minimal python3-pycurl python3-software-properties
  python3.7 python3.7-minimal readline-common software-properties-common
0 upgraded, 61 newly installed, 0 to remove and 1 not upgraded.
Need to get 8312 kB/20.7 MB of archives.
After this operation, 75.0 MB of additional disk space will be used.
Err:1 http://deb.debian.org/debian buster/main amd64 libpython3.7-minimal amd64 3.7.3-2+deb10u2
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:2 http://security.debian.org/debian-security buster/updates/main amd64 libldap-common all 2.4.47+dfsg-3+deb10u3
  404  Not Found [IP: 2a04:4e42:200::644 80]
Get:3 http://deb.debian.org/debian buster/main amd64 libexpat1 amd64 2.2.6-2+deb10u1 [106 kB]
Err:4 http://security.debian.org/debian-security buster/updates/main amd64 libldap-2.4-2 amd64 2.4.47+dfsg-3+deb10u3
  404  Not Found [IP: 2a04:4e42:200::644 80]
Err:5 http://deb.debian.org/debian buster/main amd64 python3.7-minimal amd64 3.7.3-2+deb10u2
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:6 http://deb.debian.org/debian buster/main amd64 libpython3.7-stdlib amd64 3.7.3-2+deb10u2
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:7 http://deb.debian.org/debian buster/main amd64 python3.7 amd64 3.7.3-2+deb10u2
  404  Not Found [IP: 2a04:4e42:41::644 80]
Ign:8 http://deb.debian.org/debian buster/main amd64 libsasl2-modules-db amd64 2.1.27+dfsg-1+deb10u1
Get:9 http://deb.debian.org/debian buster/main amd64 libsasl2-2 amd64 2.1.27+dfsg-1+deb10u1 [106 kB]
Ign:10 http://deb.debian.org/debian buster/main amd64 apt-transport-https all 1.8.2.1
Err:11 http://deb.debian.org/debian buster/main amd64 libkrb5support0 amd64 1.17-3
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:12 http://deb.debian.org/debian buster/main amd64 libk5crypto3 amd64 1.17-3
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:13 http://deb.debian.org/debian buster/main amd64 libkrb5-3 amd64 1.17-3
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:14 http://deb.debian.org/debian buster/main amd64 libgssapi-krb5-2 amd64 1.17-3
  404  Not Found [IP: 2a04:4e42:41::644 80]
Ign:15 http://deb.debian.org/debian buster/main amd64 libcurl4 amd64 7.64.0-4+deb10u1
Ign:16 http://deb.debian.org/debian buster/main amd64 curl amd64 7.64.0-4+deb10u1
Err:17 http://deb.debian.org/debian buster/main amd64 distro-info-data all 0.41+deb10u2
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:18 http://deb.debian.org/debian buster/main amd64 libglib2.0-0 amd64 2.58.3-2+deb10u2
  404  Not Found [IP: 2a04:4e42:41::644 80]
Ign:19 http://deb.debian.org/debian buster/main amd64 libcurl3-gnutls amd64 7.64.0-4+deb10u1
Ign:20 http://deb.debian.org/debian buster/main amd64 python-apt-common all 1.8.4.1
Ign:21 http://deb.debian.org/debian buster/main amd64 python3-apt amd64 1.8.4.1
Err:8 http://deb.debian.org/debian buster/main amd64 libsasl2-modules-db amd64 2.1.27+dfsg-1+deb10u1
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:10 http://deb.debian.org/debian buster/main amd64 apt-transport-https all 1.8.2.1
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:15 http://deb.debian.org/debian buster/main amd64 libcurl4 amd64 7.64.0-4+deb10u1
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:16 http://deb.debian.org/debian buster/main amd64 curl amd64 7.64.0-4+deb10u1
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:19 http://deb.debian.org/debian buster/main amd64 libcurl3-gnutls amd64 7.64.0-4+deb10u1
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:20 http://deb.debian.org/debian buster/main amd64 python-apt-common all 1.8.4.1
  404  Not Found [IP: 2a04:4e42:41::644 80]
Err:21 http://deb.debian.org/debian buster/main amd64 python3-apt amd64 1.8.4.1
  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/p/python3.7/libpython3.7-minimal_3.7.3-2+deb10u2_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/p/python3.7/python3.7-minimal_3.7.3-2+deb10u2_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/p/python3.7/libpython3.7-stdlib_3.7.3-2+deb10u2_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/p/python3.7/python3.7_3.7.3-2+deb10u2_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/c/cyrus-sasl2/libsasl2-modules-db_2.1.27+dfsg-1+deb10u1_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/o/openldap/libldap-common_2.4.47+dfsg-3+deb10u3_all.deb  404  Not Found [IP: 2a04:4e42:200::644 80]
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/o/openldap/libldap-2.4-2_2.4.47+dfsg-3+deb10u3_amd64.deb  404  Not Found [IP: 2a04:4e42:200::644 80]
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/a/apt/apt-transport-https_1.8.2.1_all.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/k/krb5/libkrb5support0_1.17-3_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/k/krb5/libk5crypto3_1.17-3_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/k/krb5/libkrb5-3_1.17-3_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/k/krb5/libgssapi-krb5-2_1.17-3_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/c/curl/libcurl4_7.64.0-4+deb10u1_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/c/curl/curl_7.64.0-4+deb10u1_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/d/distro-info-data/distro-info-data_0.41+deb10u2_all.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://deb.debian.org/debian/pool/main/g/glib2.0/libglib2.0-0_2.58.3-2+deb10u2_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Fetched 212 kB in 0s (771 kB/s)
Failed to fetch http://security.debian.org/debian-security/pool/updates/main/c/curl/libcurl3-gnutls_7.64.0-4+deb10u1_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/p/python-apt/python-apt-common_1.8.4.1_all.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/p/python-apt/python3-apt_1.8.4.1_amd64.deb  404  Not Found [IP: 2a04:4e42:41::644 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
--> 5abf886aad9
[1/3] STEP 3/6: RUN curl --progress-bar https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB | apt-key add -
/bin/sh: 1: curl: not found
E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation
[2/3] STEP 1/9: FROM debian:buster AS configure-mkl
Resolved "debian" as an alias (/etc/containers/registries.conf.d/000-shortnames.conf)
Trying to pull docker.io/library/debian:buster...
Error: error building at STEP "RUN curl --progress-bar https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB | apt-key add -": error while running runtime: exit status 255
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,7 @@
 ARG base=debian:buster
 
+ENV DEBIAN_FRONTEND=noninteractive
+
 ## Install official Intel MKL repository for apt
 ## Commands below adapted from:
 ##     https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo

--- a/Dockerfile
+++ b/Dockerfile
@@ -21,18 +21,14 @@ ENV GPG_FILE=/usr/share/keyrings/intel-gpg-archive-keyring.gpg
 RUN curl $GPG_URL | gpg --dearmor > $GPG_FILE
 RUN echo "deb [signed-by=$GPG_FILE] $BASE_URL/mkl all main" > /etc/apt/sources.list.d/intel-mkl.list
 
-ARG year=2020
-
-# Download debs and cache them
-RUN apt-get update && \
-    apt-get install -y -d --no-install-recommends \
-                    $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
-
 # Install build-essential
 RUN apt-get update && apt-get install -y build-essential
 
 # Install MKL
-RUN apt-get install -y --no-download --no-install-recommends \
+ARG year=2020
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
                     $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
 
 FROM $base AS configure-mkl

--- a/Dockerfile
+++ b/Dockerfile
@@ -24,7 +24,8 @@ RUN echo "deb [signed-by=$GPG_FILE] $BASE_URL/mkl all main" > /etc/apt/sources.l
 # Install MKL
 ARG year=2020
 RUN apt-get update && \
-    apt-get install -y $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
+    apt-get install -y --no-install-recommends \
+                    $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
 
 FROM $base AS configure-mkl
 COPY --from=install-mkl /opt/intel/ /opt/intel/

--- a/Dockerfile
+++ b/Dockerfile
@@ -29,7 +29,7 @@ RUN apt-get update && \
                     $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
 
 # Install build-essential
-RUN apt-get update && apt install -y build-essential
+RUN apt-get update && apt-get install -y build-essential
 
 # Install MKL
 RUN apt-get update && \

--- a/Dockerfile
+++ b/Dockerfile
@@ -21,8 +21,14 @@ ENV GPG_FILE=/usr/share/keyrings/intel-gpg-archive-keyring.gpg
 RUN curl $GPG_URL | gpg --dearmor > $GPG_FILE
 RUN echo "deb [signed-by=$GPG_FILE] $BASE_URL/mkl all main" > /etc/apt/sources.list.d/intel-mkl.list
 
-# Install MKL
 ARG year=2020
+
+# Download debs and cache them
+RUN apt-get update && \
+    apt-get install -y -d --no-install-recommends \
+                    $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
+
+# Install MKL
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
                     $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)

--- a/Dockerfile
+++ b/Dockerfile
@@ -4,11 +4,12 @@ ARG base=debian:buster
 ## Commands below adapted from:
 ##     https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo
 ##     https://github.com/eddelbuettel/mkl4deb
-FROM nobodyxu/apt-fast:latest-debian-buster-slim AS install-mkl
+FROM debian:stable-slim AS install-mkl
 
 # Install basic software for adding apt repository and downloading source code to compile
-RUN apt-auto install -y --no-install-recommends apt-transport-https ca-certificates gnupg2 gnupg-agent \
-                                                software-properties-common curl apt-utils
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends apt-transport-https ca-certificates gnupg2 gnupg-agent \
+                                               software-properties-common curl apt-utils
 
 # Add key
 RUN curl --progress-bar https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB | apt-key add -
@@ -16,7 +17,8 @@ RUN echo deb https://apt.repos.intel.com/mkl all main > /etc/apt/sources.list.d/
 
 # Install MKL
 ARG year=2020
-RUN apt-auto install -y '$(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)'
+RUN apt-get update && \
+    apt-get install -y $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
 
 FROM $base AS configure-mkl
 COPY --from=install-mkl /opt/intel/ /opt/intel/

--- a/Dockerfile
+++ b/Dockerfile
@@ -28,6 +28,9 @@ RUN apt-get update && \
     apt-get install -y -d --no-install-recommends \
                     $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
 
+# Install build-essential
+RUN apt-get update && apt install -y build-essential
+
 # Install MKL
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,7 +8,7 @@ FROM debian:stable-slim AS install-mkl
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-# Install basic software for adding apt repository and downloading source code to compile
+# Install basic software for adding apt repository
 RUN apt-get update && \
     apt-get install -y --no-install-recommends apt-transport-https ca-certificates gnupg2 gnupg-agent \
                                                software-properties-common curl apt-utils

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,12 +1,12 @@
 ARG base=debian:buster
 
-ENV DEBIAN_FRONTEND=noninteractive
-
 ## Install official Intel MKL repository for apt
 ## Commands below adapted from:
 ##     https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo
 ##     https://github.com/eddelbuettel/mkl4deb
 FROM debian:stable-slim AS install-mkl
+
+ENV DEBIAN_FRONTEND=noninteractive
 
 # Install basic software for adding apt repository and downloading source code to compile
 RUN apt-get update && \

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,8 +14,12 @@ RUN apt-get update && \
                                                software-properties-common curl apt-utils
 
 # Add key
-RUN curl --progress-bar https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB | apt-key add -
-RUN echo deb https://apt.repos.intel.com/mkl all main > /etc/apt/sources.list.d/intel-mkl.list
+ENV BASE_URL=https://apt.repos.intel.com
+ENV GPG_URL=$BASE_URL/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
+ENV GPG_FILE=/usr/share/keyrings/intel-gpg-archive-keyring.gpg
+
+RUN curl $GPG_URL | gpg --dearmor > $GPG_FILE
+RUN echo "deb [signed-by=$GPG_FILE] $BASE_URL/mkl all main" > /etc/apt/sources.list.d/intel-mkl.list
 
 # Install MKL
 ARG year=2020

--- a/Dockerfile
+++ b/Dockerfile
@@ -32,8 +32,7 @@ RUN apt-get update && \
 RUN apt-get update && apt-get install -y build-essential
 
 # Install MKL
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
+RUN apt-get install -y --no-download --no-install-recommends \
                     $(apt-cache search intel-mkl-$year | cut -d '-' -f 1,2,3,4  | tail -n 1)
 
 FROM $base AS configure-mkl

